### DISCUSSION TITLE
Warnings do not show up as warnings for custom rules

### DISCUSSION THREAD

**Discussion by rvanbekkum** [StefanMaron/BusinessCentral.LinterCop#60] [Q&A] [answered]
Hi @StefanMaron and community,

I am running into a problem with applying custom code analyzer rules in a BC build pipeline. Even though the severity is set to warning, the problems do not pop up as a warning.
Any idea of what might be causing this or how to resolve this?

See attached image, the LC-rules show up with yellow highlighting, but my custom rules do not, even though it already says "warning" :(
I am passing the paths to both LC and 'Custom' analyzers as a list (e.g., `@("...\BusinessCentral.LinterCop.dll", "...\BusinessCentral.CustomCop.dll")`) to the `-CustomCodeCops` parameter of the `Compile-AppInBcContainer` command.

![image](https://user-images.githubusercontent.com/7383241/136784666-8d23dc8f-c3e3-4883-8c52-29a8a25b4f59.png)

**Comment by rvanbekkum** [comment, marked as answer]
Found it. This line in the `BcContainerHelper` expects exactly a 2-letter code before it will convert ALC-output to AzureDevOps output:
https://github.com/microsoft/navcontainerhelper/blob/ca6472237cf13c78fa3c864d5cbb90192ed801b6/AppHandling/Convert-ALCOutputToAzureDevOps.ps1#L39

So easiest solution: switch to a 2-letter code. :)

**Reply by StefanMaron** [reply]
After I read everything like 5 times I understood whats going on :D I did not read the part where you wrote you have your own analyzer ;)

If you happen to have any rule you would like to share with the community, feel free to make a PR 😎

**Reply by rvanbekkum** [reply]
Yes, will do. 😁
